initialise/MirahezeFunctions.php

```diff
@@ -1035,6 +1036,30 @@ public static function onManageWikiCoreAddFormFields( $ceMW, $context, $dbName,
 
 		asort( $versions );
 
+		$mwSettings = new ManageWikiSettings( $dbName );
+		$setList = $mwSettings->list();
+		$formDescriptor['article-path'] = [
+			'label-message' => 'miraheze-label-managewiki-article-path',
+			'type' => 'select',
+			'options-messages' => [
+				'miraheze-label-managewiki-article-path-wiki' => '/wiki/$1',
+				'miraheze-label-managewiki-article-path-root' => '/$1',
+			],
+			'default' => $setList['wgArticlePath'] ?? '/wiki/$1',
+			'disabled' => !$permissionManager->userHasRight( $context->getUser(), 'managewiki-restricted' ),
```

Comment:
This,

Reply:
This will be restricted, there are issues with cache that sometimes lead to redirect loops when changed so they will have to remain restricted for the time being until some are done to test if it will cause problems.